### PR TITLE
Updating the introduction

### PR DESCRIPTION
**Pull request by castiz**

### CODE REVIEW

File: draft-tigress-requirements.md [hunk 1]
```diff
@@ -105,8 +105,19 @@ This document describes the use cases necessitating the secure transfer of digit
 
 # Introduction
 
-TODO Introduction
+Today, there is no standard way of transferring digital credentials securely between two devices belonging to the same platform or two different platforms. This document describes the problem space and the requirements for the solution the working group creates.
```
Comment (alexpell00) [inline draft-tigress-requirements.md:108]:
```suggestion
Today, there is no widely accepted way of transferring digital credentials securely between two devices belonging to the same platform or two different platforms. This document describes the problem space and the requirements for the solution the working group creates.
```

File: draft-tigress-requirements.md [hunk 2]
```diff
@@ -105,8 +105,19 @@ This document describes the use cases necessitating the secure transfer of digit
...
 
+# General Setting
+
+When sharing digital secure credentials, there are several actors involved. While the Tigress working group's solution will focus on sharing information between two devices and potentially an interemediary server, there are a couple more actors involved. First of all, each device communicates with its own servers. These are referenced as the OEM server or the Device OEM server. In addition, there are the companies that are providing the digital credential. These are the provisioning partners. They are in control of the provisioning information and the lifecycle of the credentials. 
+
+Without Tigress, devices are already connected with their Device OEM Servers and those servers already connect with provisioning partners. We assume that all devices follow this paradigm. 
+
+Device <-> Device OEM <-> Provisioning Partner
```
Comment (alexpell00) [inline draft-tigress-requirements.md:116]:
```suggestion
When sharing digital secure credentials, there are several actors involved. While the Tigress working group's solution will focus on sharing information between two devices, potentially through an interemediary server, there are a couple more actors involved. 

The companies that are providing the digital credential are the provisioning partners. They are in control of the provisioning information and the lifecycle of the credentials. Each device has a preexisting trust relationship between itself and the Provisioning Partner.
```
I don't think we should mention the "Device OEM" unless we  need to talk about something that only the device OEM can do. What do you think of something like this?

I think we can omit the device OEM to provisioning partner interaction, and write it off as an implementation detail.

Comment (castiz) [reply draft-tigress-requirements.md]: I added your suggestion but my last paragraph still has OEM notes. I think it is helpful, but I can remove it if you don't find it necessary!

File: draft-tigress-requirements.md [hunk 3]
```diff
@@ -105,8 +105,17 @@ This document describes the use cases necessitating the secure transfer of digit
...
+
+By having the devices communicate provisioning information to eachother, we achieve a couple goals. First, this negates the need for either device to connect directly to a provisioning partner. Devices need only connect with their OEM server rather than the N supported provisioning partners. In addition, this abstracts a user or user device from knowing if a credential is supported on the recipient device. Senders only have to send some information over to the receiver and the receiver's device handles everything else. Lastly, in combination with the requirements below, the solution the working group creates will protect user's privacy and avoid any entity except the users and provisioning partners from knowing who has shared with whom.
```
Comment (alexpell00) [inline draft-tigress-requirements.md:118]:
```suggestion
Today, there is no widely accepted way of transferring digital credentials securely between two devices belonging to the same platform or two different platforms. This document describes the problem space and the requirements for the solution the working group creates.

Tigress allows for a sender and receiver device to communicate in order to facilitate a secure credential transfer. Tigress also specifies certain privacy requirements in order to maintain a high level of user privacy.

# General Setting

When sharing digital secure credentials, there are several actors involved. While the Tigress working group's solution will focus on sharing information between two devices, potentially through an intermediary server, there are a couple more actors involved.

The companies that are providing the digital credential are the provisioning partners. They are in control of the provisioning information and the lifecycle of the credentials. Each device has a preexisting trust relationship between itself and the Provisioning Partner.

The interface between the devices and the Provisioning Partner can be proprietary or part of a public standard such as the CCC. The sender device obtains provisioning information from the provisioning partner, then shares it to the recipient device via Tigress. The recipient then takes that data and sends it to the Provisioning Partner to redeem a credential.

For some credential types the Provisioning Partner who mints new credentials is actually the sender device. In that scenario the receiver will generate new key material at the request of the sender device, and then communicate with the sender device over Tigress to its key material signed by the sender device.
```
What do you think about something like this? I removed the Device OEM portion and the section about the Device OEM providing a 1 to many interaction for the device.

**Pull request merged**